applicationinsights.ts

```diff
@@ -111,12 +115,13 @@ export function setup(setupString?: string) {
  * @returns {ApplicationInsights} this class
  */
 export function start() {
-    if(!!defaultClient) {
+    if (!!defaultClient) {
         _isStarted = true;
         _console.enable(_isConsole, _isConsoleLog);
         _exceptions.enable(_isExceptions);
         _performance.enable(_isPerformance);
         _heartbeat.enable(_isHeartBeat, defaultClient.config);
+        _webSnippet.enable(process.env.APPINSIGHTS_JAVASCRIPT_ENABLED == "true"); // Use environment variable by default
```

Comment:
This is the existing flag for legacy injection. Should we use a different one to prevent enabling both in windows?

Reply:
Having another env variable would make it harder for customers to enable this, we need to update existing IIS URL rewrite before enabling this functionality to avoid multiple injections

Reply:
Added new env variable to avoid double injection until IIS URL Rewrite feature is replaced